src/main/java/com/tabnine/TabNineProcess.java

```diff
@@ -33,7 +32,7 @@ void startTabNine() throws IOException {
         // setup since we assume it's already built into the IDE
         List<String> command = new ArrayList<>();
         command.add(TabNineFinder.getTabNinePath());
-        List<String> metadata = new ArrayList<>();
+        List<String> metadata = new ArrayList<>(asList("--no-lsp", "true"));
```

Comment:
This is silly, but no_lsp is not metadata. A metadata is a generic key=value parameter, that's passed as is to analytics.

It is better to add it to the command object, around lines 42

Reply:
OK. The separation do looks silly.